docs/2_fitting.rst

```diff
@@ -2,8 +2,636 @@
 
 Forcefield Optimization
 =======================
-The goal here is to fit a classical potential energy surface (PES) to an
-*ab-initio* PES by optimizing the classical forcefield parameters.
-This forcefield optimization is conducted using the Addaptive Rate Monte
-Carlo method described by S. Cosseddu *et al* in
-*J. Chem. Theory Comput.*, **2017**, *13*, 297–308.
+The goal of this tutorial is to obtain the classical forcefield parameters for a lead perovksite core and for the nanocrystal (NC) obtained by capping the fitted core with carboxylate ligands. In fact, the construction of a forcefield for a Quantum Dot (QD) requires parameters for a proper description of:
+
+    * the ion-ion interactions inside the nanocrystal core;
+    * the ligand anchoring group-core ions interactions at the nanocrystal surface.
+    
+The third "category" of parameters, accounting for the organic ligands, are commonly available in literature and we thus won't need to fit them.
+
+
+
+To fit the parameters, we will start from an *ab-initio* Molecular Dynamics (MD) trajectory (NVT, 300K, 5ps) of a 2.3 nm sided cubic CsPbBr_3 core (see `tutorial <https://nanotutorials.readthedocs.io/en/latest/1_build_qd.html>`_ for the Quantum Dot construction using **CAT**).
+
+Before starting the fitting we invite you to read the `documentation <https://auto-fox.readthedocs.io/en/latest/4_monte_carlo.html>`_ relative to Adaptive Rate Monte Carlo (ARMC) in Auto-FOX.
+
+The inorganic core
+---------------
```

Comment:
```suggestion
The inorganic core
-------------------
```
When a title underline is used it has to be at least as long as the title itself, otherwise Sphinx will throw a hissy fit.